spec/rubocop/cop/rspec/http_status_spec.rb

```diff
@@ -0,0 +1,55 @@
+# frozen_string_literal: true
+
+RSpec.describe RuboCop::Cop::RSpec::HttpStatus, :config do
+  subject(:cop) { described_class.new(config) }
+
+  context 'when EnforcedStyle is `symbol`' do
+    let(:cop_config) { { 'EnforcedStyle' => 'symbol' } }
+
+    it 'registers an offense when using numeric value' do
+      expect_offense(<<-RUBY)
+        it { is_expected.to have_http_status 200 }
+                            ^^^^^^^^^^^^^^^^^^^^ Prefer `symbol` over `numeric` to describe http status code.
```

Comment:
It would be nice if we could have the `^^^`’s just under the `200` value. I haven’t entirely figured out how to control this highlighting myself, so it’s not a blocker for me :-)

Reply:
Adding the offense to the `:selector` should do the trick